bin/index.js

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env node
```

Comment:
I believe this line has to stay for the bin file to actually function correctly. Did you test that using the `pandera` command still worked after this change?